docs/sources/getting-started/getting-started-elasticsearch.md

```diff
@@ -0,0 +1,107 @@
+
+### General Settings
+
+| Name   | Description                                                                           |
+| ------ | ------------------------------------------------------------------------------------- |
+| `Name` | The data source name. This is how you refer to the data source in panels and queries. |
+
+### HTTP Settings
+
+| Name     | Description                                                                                               | Value                    |
+| -------- | --------------------------------------------------------------------------------------------------------- | ------------------------ |
+| `URL`    | The IP address/hostname and optional port of your Elasticsearch instance.                                 | `http://localhost:12200` |
+| `Access` | whether to connect to your Elasticsearch instance from the browser or through Grafana's backend (Server). | `Server (default)`       |
+
+### Elasticsearch details
+
+| Name              | Description                                                                | Value                  |
+| ----------------- | -------------------------------------------------------------------------- | ---------------------- |
+| `Index name`      | The index pattern used to identify indices you want to query with grafana. | `[metrics-]YYYY.MM.DD` |
+| `Pattern`         | The time pattern for which the above index will be be interpolated.        | `Daily`                |
+| `Time field name` | The name of the field that identiefies the timestamp of your documents.    | `@timestamp`           |
+| `Version`         | The Elasticsearch version you are running.                                 | `7.0+`                 |
+
+Once you click **Save & Test** the following message will appear, confirming that Grafana is able to communicate with your Elasticsearch instance:
+
+![Elasticsearch confirmation message](/img/docs/getting-started/elasticsearch/confirmation-7-4.png)
+
+## Step 4. Create your first Elasticsearch backed dashboard
+
+1. In the Grafana side menu, hover your cursor over the **Create** (plus) icon and then click **Dashboard**.
+1. Click the **Add new panel** button.
+
+If you made Elasticsearch your default data source by toggling `default` on when creating it, it should be already selected:
+
+![Elasticsearch panel edit](/img/docs/getting-started/elasticsearch/panel-edit-7-4.png)
+
+Otherwise you can select it from the data source picker:
+
+![data source picker](/img/docs/getting-started/elasticsearch/datasource-picker-7-4.png)
+
+You can now start editing your query to get data out of your Elasticsearch instance.
+
+## Next steps
+
+Now that you have your Elasticsearch powered Grafana dashboard you can start exploring different ways of querying your Elasticsearch index to get the most out of your data.
```

Comment:
what do you mean by "links to Explore"? 
External resources (added in https://github.com/grafana/grafana/pull/30285/files#diff-33e45c92c8c56f5f72a879740aad9393cf8a259d8c073d9a8d13f4927f7702a4R107) or to "our" Explore? because this wasn't a mention to it, maybe I should reword it?